### PR TITLE
Task#6630_ui_photo_details

### PR DESCRIPTION
**Pull request by NguyenQuocBao19182**
Link Redmine : https://edu-redmine.framgia.vn/issues/6630
screenshot from 2019-02-19 12-23-58

### CODE REVIEW

**Comment by NguyenQuocBao19182** [comment]
ready

File: app/src/main/java/com/framgia/my_editor_02/screen/photoDetail/PhotoDetailFragment.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import com.framgia.my_editor_02.R;
+import com.framgia.my_editor_02.databinding.FragmentPhotoDetailBinding;
+
+public class PhotoDetailFragment extends Fragment {
+    private PhotoDetailViewModel mPhotoDetailViewModel;
+    private FragmentPhotoDetailBinding mBinding;
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/framgia/my_editor_02/screen/photoDetail/PhotoDetailFragment.java]: convert to local variable

File: app/src/main/res/layout/fragment_photo_detail.xml [hunk 2]
```diff
@@ -0,0 +1,161 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    >
+    <data>
+        <variable
+            name="photoDetailViewModel"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_photo_detail.xml]:
```suggestion
            name="viewModel"
```

File: app/src/main/res/layout/fragment_photo_detail.xml [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+            android:layout_height="@dimen/dp_300"
+            app:layout_constraintTop_toTopOf="parent"
+            />
+        <TextView
+            android:id="@+id/textViewNamePhoto"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginLeft="@dimen/dp_5"
+            android:layout_marginTop="@dimen/dp_5"
+            android:text="RayBan Success"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_photo_detail.xml]: hard code

File: app/src/main/res/layout/fragment_photo_detail.xml [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+            app:layout_constraintTop_toBottomOf="@+id/imageViewDetail"
+            />
+
+        <TextView
+            android:id="@+id/textViewUserName"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginLeft="@dimen/dp_5"
+            android:layout_marginTop="@dimen/dp_5"
+            android:text="by andwewdsdq"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_photo_detail.xml]: hard code

File: app/src/main/res/layout/fragment_photo_detail.xml [hunk 5]
```diff
@@ -0,0 +1,161 @@
...
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/textViewUserName"
+            />
+        <TextView
+            android:id="@+id/textViewLocation"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginLeft="@dimen/dp_5"
+            android:layout_marginTop="@dimen/dp_5"
+            android:text="SentaModika"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_photo_detail.xml]: hard code

**Comment by NguyenQuocBao19182** [comment]
ready

File: app/src/main/res/layout/fragment_photo_detail.xml [hunk 6]
```diff
@@ -0,0 +1,161 @@
...
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/imageViewLikes"
+            />
+        <TextView
+            android:id="@+id/textViewDowloads"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/dp_5"
+            android:layout_marginTop="@dimen/dp_5"
+            android:text="@string/_2_324_dowloads"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_photo_detail.xml]: check convention

File: app/src/main/res/layout/fragment_photo_detail.xml [hunk 7]
```diff
@@ -0,0 +1,161 @@
...
+            android:text="@string/dowload"
+            android:textColor="@color/color_white"
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintEnd_toStartOf="@+id/button3"
+            app:layout_constraintHorizontal_bias="0.5"
+            app:layout_constraintHorizontal_chainStyle="spread"
+            app:layout_constraintStart_toStartOf="parent"
+            />
+        <Button
+            android:id="@+id/button3"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_photo_detail.xml]: check convention

File: app/src/main/res/layout/fragment_photo_detail.xml [hunk 8]
```diff
@@ -0,0 +1,161 @@
...
+            android:layout_height="wrap_content"
+            android:layout_marginStart="@dimen/dp_5"
+            android:layout_marginTop="@dimen/dp_5"
+            android:text="@string/_2_324_dowloads"
+            android:textColor="@color/color_black"
+            app:layout_constraintLeft_toRightOf="@id/imageViewDowloads"
+            app:layout_constraintTop_toTopOf="@+id/imageViewDowloads"
+            />
+        <Button
+            android:id="@+id/button2"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/fragment_photo_detail.xml]: check convention

File: app/src/main/res/values/strings.xml [hunk 9]
```diff
@@ -4,4 +4,9 @@
     <string name="photos">Photos</string>
     <string name="collections">Collections</string>
     <string name="photo_editor">Photo Editor</string>
+    <string name="_1_232_views">1,232 Views</string>
+    <string name="_2_225_likes">2,225 likes</string>
+    <string name="_2_324_dowloads">2,324 dowloads</string>
```
Comment (tuanlvt-1117) [inline app/src/main/res/values/strings.xml]: không đưa data test vào

**Comment by NguyenQuocBao19182** [comment]
ready

**Comment by NguyenQuocBao19182** [comment]
ready

File: app/src/main/java/com/framgia/my_editor_02/screen/photoDetail/PhotoDetailFragment.java [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+        Bundle args = new Bundle();
+        photoDetailFragment.setArguments(args);
+        return photoDetailFragment;
+    }
+
+    @Nullable
+    @Override
+    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container,
+            @Nullable Bundle savedInstanceState) {
+        PhotoDetailViewModel mPhotoDetailViewModel = new PhotoDetailViewModel();
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/framgia/my_editor_02/screen/photoDetail/PhotoDetailFragment.java]:
```suggestion
        PhotoDetailViewModel photoDetailViewModel = new PhotoDetailViewModel();
```

File: app/src/main/java/com/framgia/my_editor_02/screen/photoDetail/PhotoDetailFragment.java [hunk 11]
```diff
@@ -0,0 +1,33 @@
...
+        FragmentPhotoDetailBinding mBinding =
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/framgia/my_editor_02/screen/photoDetail/PhotoDetailFragment.java]:
```suggestion
        FragmentPhotoDetailBinding binding =
```

**Comment by NguyenQuocBao19182** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**